org.scala-refactoring.library/src/main/scala/scala/tools/refactoring/analysis/CompilationUnitDependencies.scala

```diff
@@ -128,6 +144,21 @@ trait CompilationUnitDependencies extends CompilerApiExtensions {
     }
 
     val traverser = new TraverserWithFakedTrees {
+      var annotationTree: Option[Tree] = None
```

Comment:
I have to say, I'm not a fan of the var here. But I don't see an easy alternative (one alternative would be to thread the annotationtree through the traverse function, but would be a rather big change).